INSTALL.rst

```diff
@@ -0,0 +1,31 @@
+Installation
+============
+
+Linux
+-----
+
+PyNaCl relies on `libsodium`_, a portable C library. A copy is bundled
+with PyNaCl so to install you can run:
+
+.. code-block:: console
+
+    $ pip install pynacl
+
+If you'd prefer to use one provided by your distribution you can disable
+the bundled copy during install by running:
+
+.. code-block:: console
+
+    $ SODIUM_INSTALL=system pip install pynacl
+
+.. _libsodium: https://github.com/jedisct1/libsodium
+
+Mac OS X & Windows
+------------------
+
+PyNaCl ships as a binary wheel on OS X and Windows so all dependencies
```

Comment:
We ship a binary wheel for manylinux1 now too 😀  (but we can note that in a subsequent PR)